src/extension/react_table.js

```diff
@@ -37,41 +52,74 @@ function Table(props) {
     headerGroups,
     rows,
     prepareRow,
-  } = useTable({ columns, data })
+    allColumns,
+    getToggleHideAllColumnsProps,
+  } = useTable({ columns, data });
+
+  const toggleExpand = () => {
+    setCollapse(!collapse)
+  };
 
   return (
-    <div className="recordTable">
-      <table {...getTableProps()}>
-        <thead>
-          {headerGroups.map(headerGroup => (
-            <tr {...headerGroup.getHeaderGroupProps()}>
-              {headerGroup.headers.map(column => (
-                <th {...column.getHeaderProps()}>
-                  {column.render('Header')}
-                </th>
-              ))}
-            </tr>
-          ))}
-        </thead>
-        <tbody {...getTableBodyProps()}>
-          {rows.map(row => {
-            prepareRow(row)
-            return (
-              <tr {...row.getRowProps()}>
-                {row.cells.map(cell => {
-                  return (
-                    <td
-                      {...cell.getCellProps()}>
-                      {cell.render('Cell')}
-                    </td>
-                  )
-                })}
+    <>
```

Comment:
Are these empty `<>` correct?  If so, what do they mean?

Reply:
Thanks for the explanation.

Reply:
You are welcome!